### PR TITLE
Added drafts of the first review responses.

### LINKED ISSUE #56: Draft responses for each of the reviewers.
- [ ] Paste the responses into the F1000Research comments.

### PR DESCRIPTION
**Pull request by moorepants**
Fixes #56.

### CODE REVIEW

**Comment by moorepants** [comment]
@chrisdembia @monthubbard 

Here is a first shot at the responses. Feel free to edit.

**Comment by chrisdembia** [comment]
I'll add a commit soon.

File: review-responses/first-barre.rst [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+   et al. (2007). These indirect methods have also error as explained in Rossi
+   et al. (2013). What is the advantage of Yeadon's method compared to others?
+   A discussion on the limitations of the model proposed (joint center
+   accuracy, joint range, mass estimation, density scaling, etc.) would be also
+   of interest.
+
+We also think that a paper showing the disadvantages and advantages of these
+various models would be very interesting but we believe that this undertaking
+is out of the scope of this paper. To properly address this, would take much
+more space and time to complete than we have. We are leaving these discussions
```
Comment (chrisdembia) [inline review-responses/first-barre.rst]: I don't know if I like saying "we don't have the time".

Comment (chrisdembia) [inline review-responses/first-barre.rst]: K I'm just going to remove this sentence. I think the paragraph is fine without it.

**Comment by chrisdembia** [comment]
I am not willing to guarantee that the links in the reviews will persist as long as the paper might exist on F1000. Do you think the links to the PR's are necessary?

**Comment by chrisdembia** [comment]
Thanks a bunch for working on this. Great start, and seems like it's almost done. I made some small changes. Definitely not ready to merge yet.

**Comment by moorepants** [comment]
> I am not willing to guarantee that the links in the reviews will persist as long as the paper might exist on F1000. Do you think the links to the PR's are necessary?

You like breaking links :)

I used to think that we had to worry about broken links but I'm waning on that. It is still an issue in many cases but in the future the web will cached and people will never know a link was broken. 

I like pasting in the PR links, but we could paste in a diff instead. Or we can just say we fixed it. Doesn't really matter. I know that the second reviewer uses version control so he may find it nice.

**Comment by chrisdembia** [comment]
Ah okay good point. Let's leave them in.

**Comment by moorepants** [comment]
Is this ready?

**Pull request merged**